src/main/java/ladder/strategy/DefaultPlayResult.java

```diff
@@ -0,0 +1,28 @@
+package ladder.strategy;
+
+import ladder.domain.ladder.Ladder;
+import ladder.domain.result.ExecutionResults;
+import ladder.domain.user.LadderPlayers;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DefaultPlayResult implements PlayResult {
+
+    private final Ladder ladder;
+
+    public DefaultPlayResult(Ladder ladder) {
+        this.ladder = ladder;
+    }
+
+    @Override
+    public Map<String, String> play(LadderPlayers players, ExecutionResults results) {
+        Map<String, String> result = new HashMap<>();
+        for (int num = 0; num < players.size(); num++) {
+            int findIndex = ladder.move(num);
+            result.put(players.getPlayerNames().get(num), results.getResults().get(findIndex));
```

Comment:
```suggestion
            result.put(players.findByIndex(num), results.findByIndex(findIndex));
```
이런식으로 수정 할 수도 있을 것 같아요 ㅎㅎ

Reply:
아직 getter을 많이 쓰고있어서 계속해서 개선이 있어햐 할 것 같습니다..